ivy_tests/test_ivy/test_frontends/test_jax/test_numpy/test_statistical.py

```diff
@@ -65,6 +66,40 @@ def test_jax_einsum(
     )
 
 
+# histogram_bin_edges
+@handle_frontend_test(
+    fn_tree="jax.numpy.histogram_bin_edges",
+    values=histogram_helper(),
+)
+def test_jax_histogram_bin_edges(
+    *,
+    values,
+    on_device,
+    fn_tree,
+    frontend,
+    test_flags,
```

Comment:
I think you missed an argument here, `backend_fw` I guess. Just refer to other test implementations and your test failure outputs in the CI